shell/platform/darwin/ios/framework/Source/FlutterDartProject.mm

```diff
@@ -191,8 +191,9 @@ static BOOL DoesHardwareSupportWideGamut() {
     // Note: we intentionally pass cache_path twice. We provide two different directories
     //   to ConfigureShorebird because Android differentiates between data that persists
     //   between releases and data that does not. iOS does not make this distinction.
-    flutter::ConfigureShorebird(cache_path, cache_path, settings, shorebirdYamlContents.UTF8String,
-                                appVersion.UTF8String, appBuildNumber.UTF8String);
+    flutter::Shorebird::ConfigureShorebird(cache_path, cache_path, settings,
```

Comment:
I think namespaces are typically snake_case rather than Camel.

Reply:
`Shorebird` is a class name, but I don't think it actually needs to be a class (I believe I made this change when I was hoping I could unit test more of shorebird). Reverted.